### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by julian3493**
### Configuration

- Set up linters
- Initialize react app
- Add prop-types and big.js
- Deploy on Heroku

### CODE REVIEW

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️

Hello @julian3493,

Good job so far with your project, it is working as expected, let me list you some small changes that you need to make before moving on:

- [ ] Unnecessary files or assets, such as the React SVG Logo, are deleted. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/)) (You are missing to remove most of the default files created by react)
- [ ] App is deployed on Heroku(or other hosts like Netfly, Firebase, etc) and the live link is included at the README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/)) (Please display the same content in your local and live projects).

Please correct those issues and send another review request, good luck, and well done so far!

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
```
Comment (uche-inyama) [inline src/App.js]: delete this ``` import logo from './logo.svg'; ```

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
```
Comment (uche-inyama) [inline src/App.js]: remove the lines of code under the header tag, that is, between lines 8 to 21. 
[**NB**] I couldn't select the lines of code under the header tag.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required  :recycle: 

### Hi Julian :wave: 

I have left inline comments below, go through them and implement the changes.

Also, delete one of the CSS files.

#### Make these changes then submit another form for a code review.

#### Have a great day :slightly_smiling_face: 

##### ``` Uchechukwu Henry Inyama ```

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**